lib/Command/RecreateMasterKey.php

```diff
@@ -231,6 +231,7 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 				$this->keyManager->validateMasterKey();
 				$this->encryptAllUsers($input, $output);
 				$output->writeln("\nEncryption completed successfully\n");
+				$output->writeln("\nNOTE: Kindly log-out and log-in to access the files encrypted with new master key.");
```

Comment:
"All users will need to log out and log in again".

Not only the admin...

Reply:
Done